src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCFs.java

```diff
@@ -0,0 +1,497 @@
+            } else {
+                classifiedVariants.add(new FalsePositive(eval, FalsePositive.ReasonForFalsePositive.NO_OVERLAPPING_TRUTH));
+            }
+        }
+    }
+
+    // TODO: 6/3/18 padding can be improved
+    @Override
+    protected SVInterval getPaddedSvInterval(final VariantContext eval) {
+
+        final int homologyLength = eval.getAttributeAsInt(GATKSVVCFConstants.HOMOLOGY_LENGTH, 0);
+
+        int insTypeRightPadding = 0;
+        final String svType = eval.getAttributeAsString(GATKSVVCFConstants.SVTYPE, "");
+        if (svType.equals(GATKSVVCFConstants.SYMB_ALT_ALLELE_INS) || svType.equals(GATKSVVCFConstants.SYMB_ALT_ALLELE_DUP))
+            insTypeRightPadding = eval.getAttributeAsInt(GATKSVVCFConstants.SVLEN, 0);
+
+        // we only pad to the right with homology because we expect variants to follow left-align convention (GATK-SV does this)
+        final int leftPadding = fixedPadding;
+        final int rightPadding = fixedPadding + homologyLength + insTypeRightPadding;
+        final int chromosomeLength = refSeqDict.getSequence(eval.getContig()).getSequenceLength();
+        return new SVInterval(refSeqDict.getSequenceIndex(eval.getContig()),
+                              Math.max(0, eval.getStart() - leftPadding - 1),
+                              Math.min(eval.getEnd() + rightPadding, chromosomeLength));
+    }
+
+    private static final class SupportingTruth {
+
+        private final VariantContext truthVar;
+        private final float maxSupportScore;
+
+        SupportingTruth(final VariantContext truthVar, final float maxSupportScore) {
+            this.truthVar = truthVar;
+            this.maxSupportScore = maxSupportScore;
+        }
+    }
+
+    /**
+     * Given an eval variant and its corresponding padded interval, return concordance truth variant, if any.
+     * @param eval          variant to be evaluated
+     * @param evalInterval  padded interval constructed from {@code eval}
+     * @return              {@code null} if no concordant truth call exist
+     */
+    private SupportingTruth getConcordantTruthCalls(final VariantContext eval, final SVInterval evalInterval) {
+        SupportingTruth bestSupportingTruth = null;
+
+        final Iterator<SVIntervalTree.Entry<List<TruthVariant>>> overlappingTruths = truthTree.overlappers(evalInterval);
+
+        Float maxConcordanceScore = 0f;
+        while (overlappingTruths.hasNext()) {
+            final List<TruthVariant> truthVariants = overlappingTruths.next().getValue();
+            for (int i = 0; i < truthVariants.size(); ++i ) {
+                final TruthVariant truthVariant = truthVariants.get(i);
+                final VariantContext truthVarCtx = truthVariant.getTruth();
+                float concordanceScore = computeConcordanceScore(eval, truthVarCtx, null);
+                final Boolean concordant = variantConcordanceTester.test(eval, truthVarCtx, null);
```

Comment:
```Boolean``` or ```boolean``` I guess is just to keep consistency; I think you can change it to ```boolean``` everywhere; for example you don't check for a ```null``` before the if statement right below so there is no need for a ```Boolean``` as a ```null``` here is de-facto a bug.

Reply:
done